tests/test_cli.py

```diff
@@ -97,7 +97,7 @@ def test_set_no_file(cli):
     result = cli.invoke(dotenv_cli, ["--file", "nx_file", "set"])
 
     assert result.exit_code == 2
-    assert "Invalid value for '-f'" in result.output
+    assert "Missing argument" in result.output
```

Comment:
I set it to "Missing Argument" because it's testing the command
```dotenv --file nx_file set```
without specifying a keyword or it's value to be placed into the file. This would potentially create an empty file, and in the spirit of 
> to be conscious decision by the user
I decided to write it so that a user needed to specify at least one key value pair before it would create the file.

As to whether
```cli.invoke(dotenv_cli, ["--file", "nx_file", "set"])```
should be 
```cli.invoke(dotenv.cli.set, ["--file", "nx_file"])```

I went with the former since it is the style that the majority of the other tests were using. Both versions appear to work.

Reply:
You're right.  I was confused.